src/Illuminate/Database/Query/Builder.php

```diff
@@ -1562,7 +1564,7 @@ public function exists()
 	 * Retrieve the "count" result of the query.
 	 *
 	 * @param  string  $columns
-	 * @return int
+	 * @return mixed
```

Comment:
It should. I noticed when creating the PR, as I never look at docblocks I didn't thnk about updating it earlier. The same is true for any of the other aggregate methods (and aggregate itself).

Now that I think about it, perhaps it's better to collapse grouped results to a single value in all cases, because the behaviour now depends on having set `groupBy` or not which is kind of unpleasant. Moreover, the actual group is not accessible so all you currently have is an array of numbers for which you do not know to which group they belong, so perhaps just aggregate them manually in PHP. I will send another PR to accomplish this so you can better see what I mean.